flag/service/guest/ignition/ignition.go

```diff
@@ -0,0 +1,5 @@
+package ignition
+
+type Ignition struct {
+	IgnitionPath string
```

Comment:
It will be `--service.guest.ingnition.ingnitionpath`. I'd rather name it `Path`.